src/solver/conv_asm_implicit_gemm_gtc_bwd_nhwc.cpp

```diff
@@ -809,6 +824,13 @@ bool PerformanceConfigAsmImplicitGemmGTCBwdXdlopsNHWC::IsValid(
     if(problem.IsFp16() && gemm_k_global_split != 0 && vector_store != 1 && splits_4G > 1)
         return false;
 
+    size_t current_block_size, current_grid_size, current_splits_4G;
+    std::tie(current_block_size, current_grid_size, current_splits_4G) =
```

Comment:
Since we have got C++17 available, structure binding seems to be less wordy:
```
auto [current_block_size, current_grid_size, current_splits_4G] = GetImplicitGemmGtcDynamicBwdXdlopsNHWCKernel(problem, *this);
```
vs
```
size_t current_block_size, current_grid_size, current_splits_4G;
std::tie(current_block_size, current_grid_size, current_splits_4G) =  GetImplicitGemmGtcDynamicBwdXdlopsNHWCKernel(problem, *this);
```
But it doesn't allow to use `std::ignore`.